Makefile

```diff
@@ -35,6 +35,14 @@ local:
 	$(eval export KEY_VAULT=s189t01-gitapi-dv-loc-kv)
 	$(eval export AZ_SUBSCRIPTION=s189-teacher-services-cloud-test)
 
+.PHONY: review
+review: test-cluster
+	$(if $(PR_NUMBER), , $(error Missing environment variable "PR_NUMBER", Please specify a pr number for your review app))
+	$(eval include global_config/review.sh)
+	$(eval export DEPLOY_ENV=review)
+	$(eval export TF_VAR_pr_number=$(PR_NUMBER))
+	$(eval export TF_VAR_environment=review-pr-$(PR_NUMBER))
+
```

Comment:
statefile is -backend-config=key=${CONFIG}_aks.tfstate
so this is using review_aks.tfstate and it needs the PR number in the name